source/developers/file-transfer-protocol.rst

```diff
@@ -0,0 +1,228 @@
+  ``FTPS`` should not be confused with ``SFTP``, as ``FTPS`` does not communicate over a Secure Shell (``SSH``).
+
+  Your FTP client must support FTPS.
+
+- File Transfer Protocol (``FTP``)
+  
+  Unencrypted file transfer.
+
+  MinIO does **not** recommend using unencrypted FTP for file transfer.
+
+Supported Commands
+~~~~~~~~~~~~~~~~~~
+
+When enabled, MinIO supports the following ``ftp`` operations:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+   :width: 60%
+
+   * - FTP Client Commands
+     - Supported for MinIO
+
+   * - ``get``
+     - supported
+
+   * - ``put``
+     - supported
+
+   * - ``ls``
+     - supported
+
+   * - ``mkdir``
+     - supported
+
+   * - ``rmdir``
+     - supported
+
+   * - ``delete``
+     - supported
+
+   * - ``append``
+     - not supported
+
+   * - ``rename``
+     - not supported
+
+
+Considerations
+--------------
+
+Versioning
+~~~~~~~~~~
+
+``FTP`` and ``SFTP`` are not version-aware protocols.
+
+- If you use ``FTP`` or ``SFTP`` to get or list objects from a versioned bucket, only the latest version returns.
+- If you ``put`` an object on a versioned bucket with ``FTP`` or ``SFTP`` with the same name as an existing object, MinIO transparently creates a new version of the object, but FTP cannot be aware of this versioning work.
+
+To retrieve or work with any version of an object other than the latest version, you cannot use ``FTP`` or ``SFTP``.
+Use an S3 API Client, such as the :ref:`MinIO Client <minio-client>`.
+
+Authentication and Access
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Authentication to the MinIO deployment through the FTP protocols uses the same identity providers as regular access.
+The following user accounts can access buckets and objects through FTP protocols:
+
+- :ref:`MinIO IDP <minio-internal-idp>` users and their service accounts
+- :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` users and their service accounts
+- :ref:`OpenID/OIDC <minio-external-identity-management-openid>` service accounts
+
+Rotating credentials **cannot** access buckets or objects over FTP.
+To use rotating credentials to authenticate, you must use S3 API client or port.
+
+Authenticated users can access buckets and objects based on the :ref:`policies <minio-policy>` assigned to the user or parent user account.
+
+The FTP protocol does not require any of the ``admin:*`` `permissions <minio-policy-mc-admin-actions>`.
+The FTP protocols do not support any of the MinIO admin actions.
+
+Other MinIO Features
+~~~~~~~~~~~~~~~~~~~~
+
+The FTP protocol presents an additional means of accessing objects.
+The underlying MinIO service continues to work as expected otherwise.
+
+Other MinIO features, such as :ref:`server-side bucket replication <minio-bucket-replication-serverside>`, :ref:`site replication <minio-site-replication-overview>`, or :ref:`server-side encryption <minio-sse>`, continue to work as is.
```

Comment:
I'm not sure we need this. I think I understand the intent, but I don't think we usually make the call of "Everything works normally". 

If anything you can add at the top that enabling FTP/SFTP/FTPS does not affect any other MinIO features.

Reply:
Added at the top and removed here.